conans/client/remote_registry.py

```diff
@@ -176,9 +198,8 @@ def exists_function(remotes):
         self._add_update(remote_name, url, verify_ssl, exists_function, insert)
 
     def rename(self, remote_name, new_remote_name):
```

Comment:
(Not related to this PR, but) what if ``new_remote_name`` already exists?

Reply:
I'll add a check